app/install.php

```diff
@@ -362,9 +362,24 @@ function printStep0() {
 <?php
 }
 
+function getProcessUsername() {
+	if (function_exists('posix_getpwuid') && function_exists('posix_geteuid')) {
+		$processUser = posix_getpwuid(posix_geteuid());
+		return $processUser['name'];
+	}
+
+	exec('whoami', $output);
+	if (!empty($output)) {
```

Comment:
```suggestion
	if ($output != '') {
```
Minor: `empty()` is especially useful on structures / arrays; for scalar variables that are already existing, it is slightly overkill